### ISSUE TITLE
SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by hyenabyte** [Oak-Digital/types-4-strapi-2#28]
```
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at LoadStrapiReader.<anonymous> (/Users/hyena/Workspace/OakDigital/clients/surfnsup/surfsup/strapi/node_modules/@oak-digital/types-4-strapi-2/lib/readers/load-strapi/index.js:86:35)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/hyena/Workspace/OakDigital/clients/surfnsup/surfsup/strapi/node_modules/@oak-digital/types-4-strapi-2/lib/readers/load-strapi/index.js:6:58) [{"admin::permission":{"collectionName":"admin_permissions","info":{"name":"Permission","description
Error: Failed to parse output from remote strapi
    at LoadStrapiReader.<anonymous> (/Users/hyena/Workspace/OakDigital/clients/surfnsup/surfsup/strapi/node_modules/@oak-digital/types-4-strapi-2/lib/readers/load-strapi/index.js:90:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/hyena/Workspace/OakDigital/clients/surfnsup/surfsup/strapi/node_modules/@oak-digital/types-4-strapi-2/lib/readers/load-strapi/index.js:6:58)
```

On MacOS.

Works on NixOS on same branch

**Comment by Alexnortung** [comment]
Please check if this works in `v1.0.1`

**Comment by Alexnortung** [comment]
Fixed by #29
